CHANGELOG.md

```diff
@@ -24,6 +24,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 * `crud.len()` function to calculate the number of tuples
   in the space for memtx engine and calculate the maximum
   approximate number of tuples in the space for vinyl engine.
+* CRUD operations calculates bucket id automatically using sharding
```

Comment:
Regarding commit 'Use custom sharding key to calculate bucket id':

1. It is not clear from the commit title that is is about README and CHANGELOG files.
2. The table with functions looks more appropriate for the changelog rather than the commit, where we don't introduce any of the described functionality. Maybe other content of the commit message too.

Reply:
1. Added prefix "Doc: " to commit message.
2. Removed all content in commit message and place it to a new section "Sharding key" in README.